File: Chapter8/h.cpp
#include <iostream>

using namespace std;

const string key[10] = { 
    "nol",
    "satu",
    "dua",
    "tiga",
    "empat",
    "lima",
    "enam",
    "tujuh",
    "delapan",
    "sembilan"
};

int main() {
    int n;
    cin >> n;

    cout << key[n] << '\n';;
    return 0;
}

File: Chapter8/h.pas
var
    key: Array[0..9] of String = (
        'nol',
        'satu',
        'dua',
        'tiga',
        'empat',
        'lima',
        'enam',
        'tujuh',
        'delapan',
        'sembilan'
    );
    n: Integer;

Begin
    readln(n);
    writeln(key[n]);
End.

